strutil/strutil.go

```diff
@@ -300,13 +300,12 @@ func WordWrap(out io.Writer, text []rune, indent, indent2 string, termWidth int)
 	//       However, before this we'd count bytes instead of runes, so we'd be
 	//       even more broken. Think of it as successive approximations... at least
 	//       with this work we share tabwriter's opinion on the width of things!
-	if termWidth < 1 {
```

Comment:
probably worked only because termwidth defaulted to 40 if size was smaller than that

Reply:
it seems if padding was larger than termwidth then it got an out of bounds exception, which was not caught by this check